index.d.ts

```diff
@@ -0,0 +1,56 @@
+declare module "react-native-fcm" {
+
+    type FCMEventType = "FCMTokenRefreshed" | "FCMNotificationReceived";
+    export module FCMEvent {
+        const RefreshToken = "FCMTokenRefreshed";
+        const Notification = "FCMNotificationReceived";
+    }
+
+    export module RemoteNotificationResult {
+        const NewData = "UIBackgroundFetchResultNewData";
+        const NoData = "UIBackgroundFetchResultNoData";
+        const ResultFailed = "UIBackgroundFetchResultFailed";
+    }
+
+    export module WillPresentNotificationResult {
+        const All = "UNNotificationPresentationOptionAll";
+        const None = "UNNotificationPresentationOptionNone";
+    }
+
+    export module NotificationType {
+        const Remote = "remote_notification";
+        const NotificationResponse = "notification_response";
+        const WillPresent = "will_present_notification";
+        const Local = "local_notification";
+    }
+
+    export interface Subscription {
+        remove(): void;
+    }
+
+    export class FCM {
+        static requestPermissions(): void;
```

Comment:
Why `FCM#requestPermissions` returns `void` instead of `Boolean`?

/cc @evollu

Reply:
hmm, it should be Promise<void> actually.
check the source code, it doesn't return boolean. It rejects if failed to get permission